### PR TITLE
Code and performance improvements and bug fixes

### PR DESCRIPTION
**Pull request by rivera-ernesto**

### CODE REVIEW

File: DrawerController/Animator/DrawerAnimator.swift [hunk 1]
```diff
@@ -80,8 +80,7 @@ open class DrawerAnimator {
     
     @objc
     private func render() {
-        guard let display = displayLink else { return }
-        guard let animationClosure = animationClosure else { return }
+        guard let display = displayLink, let animationClosure = animationClosure else { return }
```
Comment (kawoou) [inline DrawerController/Animator/DrawerAnimator.swift:83]: Is there a performance advantage when changed to a single line?

File: DrawerController/DrawerController.swift [hunk 2]
```diff
@@ -225,10 +225,32 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
     }
     
     /// Actions
+    @IBAction func openLeftSide(_ sender: Any) {
+        self.openSide(.left)
```
Comment (kawoou) [inline DrawerController/DrawerController.swift]: I modified it to not use `self` when accessing inside from inside a class.
Please fix it. (and openRightSide method)

```swift
openSide(.left)
```

File: DrawerController/DrawerController.swift [hunk 3]
```diff
@@ -225,10 +225,32 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
...
+    }
+    
+    @IBAction func openRightSide(_ sender: Any) {
```
Comment (kawoou) [inline DrawerController/DrawerController.swift]: Can it be used externally even if it is not specified as public?

File: DrawerController/DrawerController.swift [hunk 4]
```diff
@@ -225,10 +225,32 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
...
+        self.openSide(.right)
+    }
+    
     public func openSide(_ side: DrawerSide, completion: (()->())? = nil) {
         /// Golden-Path
-        guard isEnable() else { return }
-        guard !isAnimating else { return }
+        guard isEnable(), !isAnimating else { return }
+        
+        if contentMap[side] == nil {
```
Comment (kawoou) [inline DrawerController/DrawerController.swift]: Lazily load 👍

File: DrawerController/Animator/DrawerAnimator.swift (hunk 1 above)
Comment (rivera-ernesto) [reply DrawerController/Animator/DrawerAnimator.swift:83]: Performance-wise not, but logically as they just return, I think it makes code clearer.

File: DrawerController/DrawerController.swift (hunk 2 above)
Comment (rivera-ernesto) [reply DrawerController/DrawerController.swift]: Well, it is better practice to always use `self.`, but sure.

File: DrawerController/DrawerController.swift (hunk 3 above)
Comment (rivera-ernesto) [reply DrawerController/DrawerController.swift]: Yes. `@IBAction` seems to make it public.

File: DrawerController/DrawerController.swift (hunk 2 above)
Comment (rivera-ernesto) [reply DrawerController/DrawerController.swift]: Updated.

**Comment by rivera-ernesto** [comment]
Updated PR.

**Review by kawoou — APPROVED** [review]

**Comment by kawoou** [comment]
I will update the version after off work.

**Pull request merged**